email-log.php

```diff
@@ -36,21 +36,74 @@
 	define( 'EMAIL_LOG_PLUGIN_FILE', __FILE__ );
 }
 
-// handle installation and table creation
+/**
+ * Handles installation and table creation.
+ */
 require_once plugin_dir_path( __FILE__ ) . 'include/install.php';
 
 /**
- * The main Plugin class
+ * The main plugin class.
+ *
+ * @since Genesis
  */
 class EmailLog {
+
+	/**
+	 * Stores the filesystem directory path with trailing slash.
+	 *
+	 * @since Genesis
+	 * @access public
```

Comment:
We should have `@access` attribute only if it either [private or protected](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods). Not for public access level.